python/src/ConstantBasisFactory_doc.i.in

```diff
@@ -17,7 +17,7 @@ A factory for constant basis of input dimension *dimension*.
 Examples
 --------
 >>> import openturns as ot
->>> basis = ConstantBasisFactory(2).build()
+>>> basis = ot.ConstantBasisFactory(2).build()
```

Comment:
@jschueller commented on this pull request:
> do you have an idea why this was not throwing an error ?

No, I agree it is strange. Either this code is not really run or something like ``from openturns import ConstantBasisFactory`` / ``from openturns import *`` was somehow run before.

Reply:
maybe its allowed when the class is from the same module, nevermind